cowprotocol/explorer
# Hardcode zero address for ipfs fetching

# Summary

Closes #169

Hard code zero address as invalid IPFS APP ID in order to avoid fetching appData info.

<img width="1211" alt="Screen Shot 2022-08-19 at 09 32 08" src="https://user-images.githubusercontent.com/11525018/185618914-74b62175-aec0-4ac6-bfcf-e98a7e8ae5a5.png">


# To Test

1. Open order details page for an specific order with appData set to zero address. (i.e `/rinkeby/orders/0x06cb5ab6d74ef0cc24602afd7ea1942c56ffb78a29fed1549c1c0624f08ea2febbd50f14aa90d2d21329367cfd6710c92b9d1fda62ddc0c3`
2. Go to AppData row and click show more button
   * You'll see an error message instantly, without fetching the info first.
